samples/button_interrupt.py

```diff
@@ -46,7 +46,8 @@ def main():
     GPIO.output(led_pin_1, GPIO.LOW)
     GPIO.output(led_pin_2, GPIO.LOW)
 
-    GPIO.add_event_detect(but_pin, GPIO.FALLING, callback=blink, bouncetime=10)
+    # By default, the poll time is 0.2 seconds, too
```

Comment:
Why is there a "too" at the end?

Reply:
I just want to make it clear that the default value and the value we are setting are the same. However, if it is not necessary, I can remove the word.

Reply:
Yeah I think you should remove it